src/components/VTextField/VTextField.js

```diff
@@ -235,10 +261,10 @@ export default {
 
       if (this.placeholder) data.domProps.placeholder = this.placeholder
 
-      if (!this.textarea && !this.multiLine) {
+      if (!this.isTextarea) {
         data.domProps.type = this.type
       } else {
-        data.domProps.rows = this.rows
+        data.domProps.rows = this._rows
```

Comment:
change to just rows, this isn't type dependent

Reply:
You mean, it should set `.rows` on an `input[type="text"]`?